hw/xbox/nv2a/nv2a_debug.c

```diff
@@ -22,18 +22,19 @@
 
 #ifdef DEBUG_NV2A_GL
 
-#include <stdio.h>
-#include <stdbool.h>
-#include <stdarg.h>
 #include <assert.h>
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdio.h>
 
-static bool has_GL_GREMEDY_frame_terminator = false;
-static bool has_GL_KHR_debug = false;
+static bool has_GL_GREMEDY_frame_terminator;
+static bool has_GL_KHR_debug;
```

Comment:
I think this change is fine. Although I did not check for accidental use of debug functions before initialization.